### DISCUSSION TITLE
connect dialog not coming up in teamtalk 5.15

### DISCUSSION THREAD

**Discussion by va3ets200** [BearWare/TeamTalk5#2132] [Q&A]
I just updated to version 5.15 of Teamtalk,  on Mac OS here.  running the  latest version of Sonoma as well on a 2020 intel Macbook air.  When I hit f2, to bring up the connect dialog, I'm finding that the dialog is not coming up at all, anything I can to do resolve this, would be appreciated.  thanks.  Dan

**Comment by bear101** [comment]
See #2130
